cegs_portal/search/templates/search/index.html

```diff
@@ -1,6 +1,16 @@
 {% extends "base.html" %}
 {% load static i18n %}
 
+{% block css %}
+{{ block.super }}
+<link href="{% static 'css/nouislider.min.css' %}" rel="stylesheet">
```

Comment:
This should be removed, it's only used on the experiment page.

Reply:
Line removed from index page